core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.controller.spec.ts

```diff
@@ -144,6 +144,12 @@ describe('Translation Modal Controller', function() {
     expect($scope.uploadingTranslation).toBe(false);
   });
 
+  it('should return null for completely completed submissions',
```

Comment:
Since this block is nested under `Translation Modal Controller`, the test name should be in the context of the modal.